java/client/src/main/java/glide/api/models/commands/stream/StreamReadOptions.java

```diff
@@ -60,6 +64,33 @@ public String[] toArgs(Map<String, String> streams) {
         return optionArgs.toArray(new String[0]);
     }
 
+    /**
+     * Converts options and the key-to-id input for {@link StreamBaseCommands#xreadBinary(Map,
+     * StreamReadOptions)} into a GlideString[].
+     *
+     * @return GlideString[]
+     */
+    public GlideString[] toArgsBinary(Map<GlideString, GlideString> streams) {
```

Comment:
same here

Reply:
Not sure how #1782 solves the issue here since we receive GS arguments to the method here and on the other PR we have no args.